compiler-core/src/type_/error.rs

```diff
@@ -521,6 +522,7 @@ pub enum UnknownValueConstructorError {
     Variable {
         name: EcoString,
         variables: Vec<EcoString>,
+        types: Vec<EcoString>,
```

Comment:
This one still has all the types here, let's use `type_with_name_in_scope` too to avoid the extra work and data.

Reply:
whoops thank you 👍🏽